CHANGELOG-developer.next.asciidoc

```diff
@@ -25,6 +25,7 @@ The list below covers the major changes between 7.0.0-rc2 and master only.
 - Build docker and kubernetes features only on supported platforms. {pull}13509[13509]
 - Need to register new processors to be used in the JS processor in their `init` functions. {pull}13509[13509]
 - The custom beat generator now uses mage instead of python, `mage GenerateCustomBeat` can be used to create a new beat, and `mage vendorUpdate` to update the vendored libbeat in a custom beat. {pull}13610[13610]
+- Altered all remaining uses of mapval to use the renamed and enhanced version: [go-lookslike](https://github.com/elastic/go-lookslike) instead, which is a separate project. The mapval tree is now gone. {pull}14165[14165]
```

Comment:
Just checking if this is asciidoc syntax? _Lookslike_ markdown.

Reply:
Ah, good catch, will fix!